ui/components/app/transaction-detail/index.scss

```
@@ -15,4 +15,55 @@
       text-transform: uppercase;
     }
   }
+
+  &-edit-V2 {
+    margin-bottom: 10px;
+    display: flex;
+    align-items: baseline;
+    justify-content: flex-end;
+    padding-top: 20px;
```

See 8px grid comment
```suggestion
    padding-top: 16px;
```
Or `24px` whichever suits